src/DatasetBrowser/DatasetBrowserTable/index.jsx

```diff
@@ -2,32 +2,42 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReactTable from 'react-table';
 import 'react-table/react-table.css';
-import { GuppyConfigType, TableConfigType } from '../configTypeDef';
+// import { GuppyConfigType, TableConfigType } from '../configTypeDef';
 import { capitalizeFirstLetter } from '../../utils';
-import './ExplorerTable.css';
+import './DatasetBrowserTable.css';
+import IconicLink from '../../components/buttons/IconicLink';
 import LockIcon from '../../img/icons/lock.svg';
 
-class ExplorerTable extends React.Component {
+function truncateTextIfNecessary(text) {
+  if (!text || text.length < 405) {
+    return text;
+  }
+  return text.slice(0, 405) + '...';
+}
+
+class DatasetBrowserTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       loading: false,
       pageSize: props.defaultPageSize,
       currentPage: 0,
+      filteredData: this.props.filteredData,
+      paginatedData: this.props.filteredData,
     };
   }
 
   getWidthForColumn = (field, columnName) => {
```

Comment:
might need to change something here or in the css to make the table fit in page width? Currently the button only get displayed part of it and you need to scroll right to see it all.

Reply:
Yeah good point! I altered the table to fix this problem. LMK if it looks ok

Reply:
has this change been deployed to qa-test3?